lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -125,55 +125,54 @@ protected function processResponseOutput( \MWHttpRequest $request ) {
 	 *
 	 * @throws ClientException
 	 */
-	public function login( $username, $password )
-	{
+	public function login( $username, $password ) {
 		// Convert the array to URL-encoded query string, so the Content-Type
 		// for the POST request is application/x-www-form-urlencoded.
 		// It would be multipart/form-data which is not supported
 		// by the Helios service.
-		$postData = http_build_query([
-			'username'	=> $username,
-			'password'	=> $password
-		]);
+		$postData = http_build_query( [
+			'username' => $username,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "username" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)